libs/designsystem/src/lib/components/modal/services/modal.helper.ts

```diff
@@ -14,11 +15,16 @@ export class ModalHelper {
   // TODO: Make presentingElement an instance field when
   // forRoot()/singleton services has been solved:
   private static presentingElement: HTMLElement = undefined;
+  private renderer: Renderer2;
 
   constructor(
     private ionicModalController: ModalController,
-    private modalAnimationBuilder: ModalAnimationBuilderService
-  ) {}
+    private modalAnimationBuilder: ModalAnimationBuilderService,
+    private rendererFactory: RendererFactory2,
+    @Inject(DOCUMENT) private document: any
```

Comment:
Please use specific type:
```suggestion
    @Inject(DOCUMENT) private document: Document
```

Reply:
Cant declare it as type Document in constructor, due to `strictMetadataEmit` in tsconfig.json. See [here for more information](https://github.com/angular/angular/issues/20351l)